ROS/controls/src/nuc_motor_controls.cpp

```diff
@@ -0,0 +1,89 @@
+#include "ros/ros.h"
```

Comment:
Please rework the C++ script and header to compile: catkin_make fails to create an executable of this file.